### DISCUSSION TITLE
New ruin guardians seem to be desperately needing more variants for balance reasons.

### DISCUSSION THREAD

**Discussion by SpazzMaticus107** [FakeFishGames/Barotrauma#7005] [Unstable feedback]
The new ruin guardians deal little to no damage to players wearing pucs, not to mention players with pucs AND hazmat/boiler suits. I think sooner later we could see a lot of players wearing a combination of these items to completely cheese through any alien ruins mission.

Here's some hand variants that could potentially help in making the guardians more relevant.

- melee alien stunbaton guardian hand (blunt trauma, stun)

- melee alien blade hand variant (bleeding, lacerations)

- ranged alien ancient weapon hand variant (burn, radiation)

- ranged alien stungun hand variant (radiation, stun)

- ranged alien pulse laser hand variant (burn, radiation)

This post is also a loveletter to radiation afflicition that would fit alien theme very well. Radiation damage type is nothing to be afraid of. All doctors who craft their own medicine know how common stabilozine is in medical recipes and always have lots of it on hand. This change would be a wonderfull addition to spice things up with the new alien ruins update.

**Comment by provet1** [comment]
Please don't use abbreviations without clarifying what it means. I've no idea what a "pucs" is. I assume it's a diving suits?

**Reply by SpazzMaticus107** [reply]
Im raising my concerns about a diving suit called pucs, that is to go along with the other diving suit such as combat diving suit and the abyss diving suit in the new update. Im not using the full name od pucs simply because other unstable testers and also devs (to whom this post is directed to) will immediately know what im taking about.

**Reply by James-E-A** [reply]
"PUCS" is the in-game name of the new specialized diving suit variant craftable by certain mid-to-late-game Electrical Engineers

https://barotraumagame.com/wiki/PUCS
